cpp/memilio/epidemiology/state_age_function.h

```diff
@@ -58,6 +58,12 @@ namespace mio
  * decreasing. This is no limitation as the support is only needed for StateAgeFunctions of Type a) as given above.
  * For classes of type b) a dummy implementation logging an error and returning -2 for get_support_max() should be implemented.
  *
+ * The get_mean method is virtual and implements a basic version to determine the mean value of the StateAgeFunction. 
+ * The base class implementation uses the fact that the StateAgeFunction is a survival function 
```

Comment:
Not every StateAgeFunction is a survival function, only StateAgeFunctions of type a). If the get_mean method relies on the property that StateAgeFunction is a survival function and can only be used for StateAgeFunctions of type a), this should be mentioned here.